code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -546,7 +546,7 @@ ADMIN_VERB(centcom_podlauncher, R_ADMIN, "Config/Launch Supplypod", "Configure a
 
 /datum/centcom_podlauncher/ui_close(mob/user) //Uses the destroy() proc. When the user closes the UI, we clean up the temp_pod and supplypod_selector variables.
 	QDEL_NULL(temp_pod)
-	QDEL_NULL(cam_screen)
+	cam_screen.hide_from(user)
 	qdel(src)
```

Comment:
How is this different? 

`qdel(src)` calls `QDEL_NULL(cam_screen)` anyways which in turn calls `hide_from`. Am I missing something?

Reply:
I think that `QDEL_NULL(cam_screen)` does not call `hide_from`. It only qdels cam_screen and sets it to null. This probably changes nothing, but I think there is a reason why most `screens` are hidden before deletion

Reply:
?
https://github.com/tgstation/tgstation/blob/522963a0197c893c8117be4b4d59e2f518c8e0b2/code/_onclick/hud/map_view.dm#L17-L21

Reply:
whoops